### ISSUE TITLE
Gibson test set?

### ISSUE THREAD

**Issue by ericchen321** [facebookresearch/habitat-lab#651]
## Habitat-Lab and Habitat-Sim versions
Habitat-Lab: master

Habitat-Sim: master

## Docs and Tutorials
Did you read the docs? https://aihabitat.org/docs/habitat-lab/
Yes

Did you check out the tutorials? https://aihabitat.org/tutorial/2020/
Yes

## ❓ Questions and Help
Hello, we are trying to reproduce experiment results in Table 2 from the [ICCV 2019 paper](https://arxiv.org/pdf/1904.01201.pdf). 

Specifically we are wondering if the Gibson test set referred in the Table's caption is the same as the `val/` split from `pointnav_gibson_v1.zip`? The zip file seems to contain only three splits - `train/`, `val/` and `val_mini/`, so we are wondering which split contains the episode definitions for this experiment.

**Comment by erikwijmans** [comment]
No, the gibson test set is private as we use it for challenges.  We aren't running a pointnav challenge for the 2019 setting anymore, so please use just the validation set.

**Comment by ericchen321** [comment]
Thanks!
